deployment/network-operator/values.yaml

```diff
@@ -181,6 +185,29 @@ nic-configuration-operator-chart:
       name: nic-configuration-operator-daemon
       tag: v0.1.14
 
+# Maintenance Operator chart related values.
+maintenance-operator-chart:
+  operator:
+    image:
+      repository: gghcr.io/mellanox
```

Comment:
typo here, use make release-build to propagate the changes from hack/release.yaml

Reply:
Done